src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java

```diff
@@ -0,0 +1,214 @@
+        final List<String> list = settings.getAsList(key, defaultList);
+        if (list.size() == 1 && "NONE".equals(ignoreCaseForNone? list.get(0).toUpperCase() : list.get(0))) {
+            return Collections.emptyList();
+        }
+        return list;
+    }
+
+    public boolean isRestApiAuditEnabled() {
+        return isRestApiAuditEnabled;
+    }
+
+    public boolean isTransportApiAuditEnabled() {
+        return isTransportApiAuditEnabled;
+    }
+
+    public boolean shouldResolveBulkRequests() {
+        return resolveBulkRequests;
+    }
+
+    public boolean shouldLogRequestBody() {
+        return logRequestBody;
+    }
+
+    public boolean shouldResolveIndices() {
+        return resolveIndices;
+    }
+
+    public boolean shouldExcludeSensitiveHeaders() {
+        return excludeSensitiveHeaders;
+    }
+
+    public Set<String> getIgnoredAuditUsers() {
+        return ignoredAuditUsers;
+    }
+
+    public Set<String> getIgnoredComplianceUsersForRead() {
+        return ignoredComplianceUsersForRead;
+    }
+
+    public Set<String> getIgnoredComplianceUsersForWrite() {
+        return ignoredComplianceUsersForWrite;
+    }
+
+    public Set<String> getIgnoredAuditRequests() {
+        return ignoreAuditRequests;
+    }
+
+    public EnumSet<AuditCategory> getDisabledRestCategories() {
+        return disabledRestCategories;
+    }
+
+    public EnumSet<AuditCategory> getDisabledTransportCategories() {
+        return disabledTransportCategories;
+    }
+
+    public String getOpendistrosecurityIndex() {
+        return opendistrosecurityIndex;
+    }
+
+    public void log(Logger logger) {
+        logger.info("Auditing on REST API is {}.", isRestApiAuditEnabled ? "enabled" : "disabled");
+        logger.info("Auditing on Transport API is {}.", isTransportApiAuditEnabled ? "enabled" : "disabled");
+        logger.info("Auditing of request body is {}.", logRequestBody ? "enabled" : "disabled");
+        logger.info("Bulk requests resolution is {} during request auditing.", resolveBulkRequests ? "enabled" : "disabled");
+        logger.info("Index resolution is {} during request auditing.", resolveIndices ? "enabled" : "disabled");
+        logger.info("Sensitive headers auditing is {}.", excludeSensitiveHeaders ? "enabled" : "disabled");
+        logger.info("Auditing requests from {} users is disabled.", ignoredAuditUsers);
+        logger.info("Compliance read operation requests auditing from {} users is disabled.", ignoredComplianceUsersForRead);
+        logger.info("Compliance write operation requests auditing from {} users is disabled.", ignoredComplianceUsersForWrite);
+        logger.info("{} are excluded from REST API auditing.", disabledRestCategories);
+        logger.info("{} are excluded from Transport API auditing.", disabledTransportCategories);
+        logger.info("Open distro auditing uses {} index(alias?) to write auditing events.", opendistrosecurityIndex);
```

Comment:
My bad for not being clear. I was not sure if `opendistrosecurityIndex` can be an alias instead of an index, so I put "alias?". If it can be an alias, please use "index or alias", otherwise just "index".

Reply:
It uses that index to monitor change events for internal configuration.